docs/user-guide/connectors/custom-connectors.rst

```diff
@@ -26,10 +26,11 @@ To send a message, you would run a command like:
 .. code-block:: bash
 
     curl -XPOST http://localhost:5000/webhooks/myio/webhook \
-      -d '{"sender": "user1", "message": "hello"}' \
+      -d '{"sender": "user1", "message": "hello", "metadata": {"type":"text"}}' \
```

Comment:
Yes, there was the first place that I wanted to add, but all the code is imported from `channel.py`:
```
.. literalinclude:: ../../../rasa/core/channels/channel.py
   :pyobject: RestInput
```
So I decided to add there. But looking now, I agree it's not a good place.
@erohmensing Do you have any suggestion?
I guess it could be a good option add another `curl` example explaining in a paragraph the `metadata` after the first example.

Reply:
Hmm, yeah not quite sure where to put this really. Maybe just add more info underneath the example under a header like "handling metadata" where you say essentially, with a custom connector, you can send extra information from your front-end all the way through to your action server so that you can use e.g. the user's name or location in your actions. and then show an example of a request that you'd send